docs/whatsnew.rst

```diff
@@ -13,23 +13,31 @@ API-breaking changes
 ^^^^^^^^^^^^^^^^^^^^
 - The ``yank.mpi`` module and the objects in the ``yank.multistate`` package, which were deprecated in 0.24.0, have now
   been removed, and they can be found in the ``mpiplus`` and ``openmmtools`` libraries respectively.
+- The function ``yank.pipeline.trailblaze_alchemical_protocol`` has been renamed
+  ``yank.pipeline.run_thermodynamic_trailblazing`` (`#1180 <https://github.com/choderalab/yank/pull/1180>`_).
 
 New features
 ^^^^^^^^^^^^
-- The trailblaze algorithm used for the authomatic generation of the alchemical path is now capable of resuming after an
-  unexpected interruption or crash. The samples generated during the process are used to initialize the replicas of the
-  replica exchange or SAMS free energy calculation (`#1176 <https://github.com/choderalab/yank/pull/1176>`_).
+- The thermodynamic trailblazing algorithm used for the authomatic generation of the alchemical path is now capable of
```

Comment:
Could you link to the part of the docs that describes this option?

Reply:
Will do, thanks!